src/rust/sdk/src/lib.rs

```diff
@@ -187,13 +196,23 @@ impl EconiaClient {
             .map(|r| r.is_some())
     }
 
+    /// Checks if the user is registered for a coin.
+    ///
+    /// # Arguments:
+    ///
+    /// * `coin` - Aptos [TypeTag] for the given coin.
     pub async fn is_registered_for_coin(&self, coin: &TypeTag) -> EconiaResult<bool> {
         let coin_store = format!("0x1::coin::CoinStore<{}>", coin);
         self.fetch_resource(self.user_account.address(), &coin_store)
             .await
             .map(|r| r.is_some())
     }
 
+    /// Returns the user's coin balance.
+    ///
+    /// # Arguments:
+    ///
+    /// * `coin` - Aptos [TypeTag] for the given coin.
```

Comment:
Again here we might want to disambiguate versus a `MarketAccount` because a user's `CoinStore` and `MarketAccount` balances are independent